.travis.yml

```diff
@@ -36,6 +36,15 @@ matrix:
         - pycodestyle scipy benchmarks/benchmarks
         - |
           grep -rlIP '[^\x00-\x7F]' scipy | grep '\.pyx\?' | sort > unicode.out; grep -rlI '# -\*- coding: \(utf-8\|latin-1\) -\*-' scipy | grep '\.pyx\?' | sort > coding.out; comm -23 unicode.out coding.out > test_code.out; cat test_code.out;  test \! -s test_code.out
+    - python: 3.7
+      if: type = pull_request
+      arch: arm64
+      dist: bionic
+      env:
+        - TESTMODE=fast
+        - COVERAGE=
+        - USE_SDIST=1
+        - PIP_NO_CACHE_DIR=off
```

Comment:
This env var exists but is basically undocumented. If I read it correctly, this _disables turning off_, so turns the cache on - which is the default I think?

Reply:
@rgommers I have added this var because on arm64 as I was getting the below issue in order to resolve the same I have added this: 

```
 WARNING: Building wheel for scipy failed: [Errno 13] Permission denied: '/home/travis/.cache/pip/wheels/37'
Failed to build scipy
Cleaning up...
  Removing source in /tmp/pip-req-build-jfbnl8mm
Removed build tracker: '/tmp/pip-req-tracker-95zki2iq'
ERROR: Could not build wheels for scipy which use PEP 517 and cannot be installed directly
Exception information:
Traceback (most recent call last):
  File "/home/travis/miniconda3/lib/python3.7/site-packages/pip/_internal/cli/base_command.py", line 186, in _main
    status = self.run(options, args)
  File "/home/travis/miniconda3/lib/python3.7/site-packages/pip/_internal/commands/install.py", line 375, in run
    ", ".join(r.name for r in pep517_build_failures)))
pip._internal.exceptions.InstallationError: Could not build wheels for scipy which use PEP 517 and cannot be installed directly
```

Reply:
Interesting, looks like a TravisCI bug. Thanks for explaining @odidev